src/controllers/contactadmin.js

```diff
@@ -0,0 +1,34 @@
+const createAdminModel = require("../models/contactadmin");
+const { db } = require("../databaseSetup");
+const { DataTypes } = require("sequelize");
+
+const ContactAdmin = createAdminModel(db, DataTypes);
+
+const getAdmins = async (req, res) => {
+  const admins = await ContactAdmin.findAll();
+  res.json(admins);
+};
+
+const getAdminById = async (req, res) => {
+  const admin = await ContactAdmin.findAll({
```

Comment:
Please replace it with findOne method.
Because for now it always returns array with one element instead of just one object

Reply:
Done.